src/components/ChatItem.tsx

```diff
@@ -0,0 +1,55 @@
+import styled from 'styled-components';
+import { useCallback, useState, useEffect } from 'react';
+import { IChatTypes } from '../Recoil/chat';
+import { useRecoilState, useRecoilValue } from 'recoil';
+import Ron from '../images/Ron.jpeg';
+import Malfoy from '../images/Malfoy.jpeg';
+
+const ChatBox = styled.div`
+  display: flex;
+  justify-content: flex-start;
+`;
+const TextBox = styled.div<{ isActive: boolean }>`
+  background-color: ${(props) =>
+    props.isActive === true ? '#FAF55C' : '#F7DEF7'};
+  color: black;
+  height: 1.5rem;
+  font-size: 15px;
+  width: max-content;
+  margin-left: ${(props) => (props.isActive === true ? 'auto' : '7px')};
+  margin-right: 10px;
+  margin-top: 17px;
+  margin-bottom: 3px;
+  padding: 5px 7px 3px 6px;
+  border-radius: 3%;
+  opacity: 0.8;
+`;
+const Profile = styled.div<{ isActive_1: boolean; isActive_2: boolean }>`
+  width: 3.5rem;
+  height: 3.5rem;
+  border-radius: 100%;
+  background-position: center;
+  margin: 5px 10px 3px 10px;
+  background-image: url(${(props) =>
+    props.isActive_2 === true ? Ron : Malfoy});
+  background-size: cover;
+  border: solid 1px #ffccff;
+  opacity: 0.9;
+  display: ${(props) => (props.isActive_1 === true ? 'none' : 'flex')};
+`;
```

Comment:
```suggestion
const Profile = styled.div<{ isActive_1?: string; isActive_2?: string}>`
  background-image: url(${(props) =>
    props.isActive_2 || Malfoy});
  display: ${(props) => (props.isActive_1 || 'flex')};
`;
```

`boolean` 값 말고 이렇게도 `props`를 사용하실 수 있습니다! 참고하시라고 남겨놓습니다

+) isActive_1 / 2 보다 좀 더 직관적으로 이해할 수 있는 이름으로 사용해봅시다! ex) `display`, `imgName`